src/loki/worker.cc

```diff
@@ -242,6 +242,7 @@ loki_worker_t::loki_worker_t(const boost::property_tree::ptree& config,
   max_best_paths = config.get<unsigned int>("service_limits.trace.max_best_paths");
   max_best_paths_shape = config.get<size_t>("service_limits.trace.max_best_paths_shape");
   max_alternates = config.get<unsigned int>("service_limits.max_alternates");
+  allow_verbose = config.get<bool>("allow_verbose", false);
```

Comment:
IMO it's fine to allow for a non-existing key in the valhalla.json here, what do you think?

Reply:
yep we have a few like that, shoudl the key be `service_limits.status.allow_verbose`?

Reply:
jup